ios/app/PrivacyInfo.xcprivacy

```diff
@@ -17,6 +17,8 @@
 			<string>NSPrivacyAccessedAPICategoryUserDefaults</string>
 			<key>NSPrivacyAccessedAPITypeReasons</key>
 			<array>
+				<string>1C8F.1</string>
```

Comment:
Why are these needed now?

Reply:
These are defaults, it seems. 
https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api#4278401

Reply:
👍